kubernetes_state/test/test_kubernetes_state.py

```diff
@@ -188,6 +198,39 @@ def test__update_kube_state_metrics_v040(self, mock_poll):
                 self.assertMetric(metric)
 
         self.assert_resourcequota()
+    
+    @mock.patch('checks.prometheus_check.PrometheusCheck.poll')
+    def test__join_custom_labels(self, mock_poll):
+        f_name = os.path.join(os.path.dirname(__file__), 'ci', 'fixtures', 'prometheus', 'prometheus.txt')
+        with open(f_name, 'rb') as f:
+            mock_poll.return_value = MockResponse(f.read(), 'text/plain')
+
+        config = {
+            'instances': [
+                {
+                'host': 'foo',
+                'kube_state_url': 'http://foo',
+                'label_joins': {
+                    'kube_deployment_labels': {
+                        'label_to_match': 'deployment',
+                        'labels_to_get':['label_addonmanager_kubernetes_io_mode']
+                    }
+                },
+            }]
+        }
+        # run check twice to have pod/node mapping
```

Comment:
you can rephrase the comment to say label mapping, because it was specific to another test